Sources/Workspace/Workspace.swift

```diff
@@ -3661,7 +3661,7 @@ extension FileSystem {
 extension Workspace.Location {
     func validatingSharedLocations(
         fileSystem: FileSystem,
-        warningHandler: (String) -> Void
+        warningHandler: @escaping (String) -> Void
```

Comment:
Didn't find a way to convey that my helper closure at the call site is non-escaping, that's the reason for this change. I might be missing something here.